README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://ci.vmware.run/api/badges/vmware/vic/status.svg)](https://ci.vmware.run/vmware/vic) [![Coverage Status](https://coverage.vmware.run/badges/vmware/vic/coverage.svg)](https://coverage.vmware.run/vmware/vic) [![Download](https://api.bintray.com/packages/vmware/vic/Download/images/download.svg)](https://bintray.com/vmware/vic/Download/_latestVersion) [![Go Report Card](https://goreportcard.com/badge/github.com/vmware/vic)](https://goreportcard.com/report/github.com/vmware/vic)
+[![Build Status](https://ci.vmware.run/api/badges/vmware/vic/status.svg)](https://ci.vmware.run/vmware/vic) [![Coverage Status](https://coverage.vmware.run/badges/vmware/vic/coverage.svg)](https://coverage.vmware.run/vmware/vic) [![Download](https://api.bintray.com/packages/vmware/vic/Download/images/download.svg)](https://bintray.com/vmware/vic/Download/_latestVersion) [![Go Report Card](https://goreportcard.com/badge/github.com/vmware/vic)](https://goreportcard.com/report/github.com/vmware/vic)[![Swagger Validator](http://online.swagger.io/validator/debug?url=https://raw.githubusercontent.com/vmware/vic/master/lib/apiservers/portlayer/swagger.json)
```

Comment:
I think http://online.swagger.io/validator?url=https://raw.githubusercontent.com/vmware/vic/master/lib/apiservers/portlayer/swagger.json is the correct url, yours has an extra debug